packages/dev/core/src/Cameras/camera.ts

```diff
@@ -620,7 +636,15 @@ export class Camera extends Node {
                 this._cache.aspectRatio === engine.getAspectRatio(this) &&
                 this._cache.projectionPlaneTilt === this.projectionPlaneTilt;
         } else {
+            if (this.oblique) {
+                check =
+                    this._cache.oblique === this.oblique ||

```

Comment:
If the values inside the `this.oblique` object change but not the object itself, this line will set `check` to `true` whereas it shouldn't.

You could use `&&` instead of `||`, but it could be possible that `_cache.oblique` is not the same than `this.oblique` but that the values inside these objects are the same (in which case the check should be ok)...

I think you should not use an `oblique` instance in the `cache` object but just add the 3 properties directly. It will make for a simpler test here.

Reply:
Okay, I'll make the change

Reply:
Changed